pkgs/open-policy-agent/gatekeeper/registry.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
+packages:
+  - type: github_release
+    repo_owner: open-policy-agent
+    repo_name: gatekeeper
+    description: "Gatekeeper - Policy Controller for Kubernetes"
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: Version == "v3.12.0-rc.0"
+        no_asset: true
```

Comment:
I get an error event hough I have set `version_constraint` 🤔 
```
ERRO[0000] install the package                           aqua_version=2.48.2-1 env=linux/amd64 error="no asset is released for this version" package_name=open-policy-agent/gatekeeper package_version=v3.12.0-rc.0 program=aqua registry=standard
```

Reply:
This is totally expected.
You should remove this version from pkg.yaml.

Reply:
It removed on below commit:
https://github.com/aquaproj/aqua-registry/pull/34373/commits/84e311db3aed7e15236bb146600516a251a422ca